lib/routes/routeBackbeat.js

```diff
@@ -1615,6 +1648,10 @@ function routeBackbeat(clientIP, request, response, log) {
             return standardMetadataValidateBucketAndObj(mdValParams, request.actionImplicitDenies, log, next);
         },
         (bucketInfo, objMd, next) => {
+            // function already called
```

Comment:
* comment not clear
* this also hints that this function should not be in the same waterfall : this function should only be called to finish the `standardMetadataValidateBucketAndObj` branch, and not in the earlier branch from line 1607